pytorch_lightning/trainer/trainer.py

```diff
@@ -1585,7 +1584,7 @@ def _call_ttp_hook(
         if not callable(fn):
             return
 
-        with self.profiler.profile(hook_name):
+        with self.profiler.profile(f"[TrainingTypePlugin]{self.training_type_plugin.__class__.__name__}.{hook_name}"):
```

Comment:
Use `Strategy` already insteaad of `TrainingTypePlugin`. Avoids follow-up changes.